### PR TITLE
Add more types

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add more types

## Links to any relevant issues

#42

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Running examples

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/examples/wallet/12-prepare_output.py [hunk 1]
```diff
@@ -17,17 +17,9 @@
...
-        "unlocks":
-            {
-                "expirationUnixTime": 1676570528,
-            },
-    }
-)
-print(f"Output: {json.dumps(output, indent=4)}")
+output = account.prepare_output(OutputParams(
+    "rms1qprutadk4uc9rp6h7wh7sech42sl0z40ztpgyygr5tf0cn5jrqshgm8y43d", "1000000", unlocks=Unlocks(expirationUnixTime=1676570528)))
+print(f"Output: {json.dumps(output.as_dict(), indent=4)}")
```
Comment (thibault-martinez) [inline bindings/python/examples/wallet/12-prepare_output.py:22]: json dumps still necessary?

File: bindings/python/iota_sdk/types/output_params.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+class ReturnStrategy(str, Enum):
+    """Return strategy for the StorageDeposit.
+    """
+    Return = 'Return'
+    Gift = 'Gift'
+
+
+@dataclass
+class StorageDeposit():
+    """Unlocks for OutputParams.
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/types/output_params.py]: no

File: bindings/python/examples/wallet/12-prepare_output.py (hunk 1 above)
Comment (Thoralf-M) [reply bindings/python/examples/wallet/12-prepare_output.py:22]: To have the output with the other languages aligned yes

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

File: bindings/python/iota_sdk/types/network_info.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+from __future__ import annotations
+from dataclasses import dataclass
+from iota_sdk.types.node_info import NodeInfoProtocol
+
+
+@dataclass
+class NetworkInfo:
+    """Network and PoW related information.
+    """
+
```
Comment (qrayven) [inline bindings/python/iota_sdk/types/network_info.py:13]: Total nit, sometimes you have a new line after the description of the class, and sometimes you haven't

Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/network_info.py:13]: Should all be solved when we have a format CI 😊

**Pull request merged**